### DISCUSSION TITLE
Crash investigation

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#2108] [Q&A]
- Hypothesis: yimmenu is setting off an automated crash response from other menu(s)

- Some logs: 
[redacted_cout.log](https://github.com/YimMenu/YimMenu/files/12566284/redacted_cout.log)
[redacted_09-09-2023-04-40-20_cout.log](https://github.com/YimMenu/YimMenu/files/12566285/redacted_09-09-2023-04-40-20_cout.log)
[redacted_09-08-2023-18-21-37_cout.log](https://github.com/YimMenu/YimMenu/files/12566286/redacted_09-08-2023-18-21-37_cout.log)
[redacted_09-08-2023-17-14-34_cout.log](https://github.com/YimMenu/YimMenu/files/12566287/redacted_09-08-2023-17-14-34_cout.log)
[redacted_09-08-2023-07-20-56_cout.log](https://github.com/YimMenu/YimMenu/files/12566288/redacted_09-08-2023-07-20-56_cout.log)

- Conditions present during each crash: 
1. Yimmenu user was not host and playing with others
2. Another user was nearby (fairly close or in same vehicle)
3. Yimmenu user was driving a vehicle
4. Yimmenu user's vehicle hit something destructible (fence, a body on the road, another car, a road sign, etc)
5. Yimmenu user was crashed after impact with destructible object

- Help needed: testers with menus that have crash capability
I would like to know if anyone would be willing to test whether yimmenu is provoking a crash response from other menus in these conditions. 

And if you think I'm mistaken then any other points of view are welcome. The goal is to figure out what is causing the crash, not whether my suspicions are right. Any help would be greatly appreciated.

**Comment by ghost** [comment]


**Comment by ghost** [comment]


**Reply by xiaoxiao921** [reply]
The big issue is that you kept talking about those but if you were doing minimal work you would've seen in plenty of issues that it was reported properly already through either the screenshot of the windows popup / messagebox or log files, meanwhile you spammed a lot of comments without any log or screenshot in different unrelated issues
